TESTING/LIN/cerrtr.f

```diff
@@ -70,7 +70,7 @@ SUBROUTINE CERRTR( PATH, NUNIT )
 *     .. Local Scalars ..
       CHARACTER*2        C2
       INTEGER            INFO
-      REAL               RCOND, SCALE
+      REAL               RCOND, SCALE, SCALES(0)
```

Comment:
Pardon my ignorance - is the zero-size dimension (here and in the other ?errtr files) intentional ? This is documented (and used) as an "output" parameter of the respective ?clatrs3 function in which call it is occurs, and AFAIK assigning to a zero-sized array is undefined behavior at best...

Reply:
Sorry, I misread - NRHS is guaranteed to be at most zero in all cases here so ?latrs3 will do an early exit without writing to scale(s).

Reply:
Yeah, that was my thinking. All the calls are with `NRHS <= 0` and the documentation says `SCALE` is of dimension `NRHS` (without an extra restriction like `MAX(1, NRHS)` that other functions have), so `0` seemed like the most natural dimension for the argument to me. Happy for it to be changed though if you think other dimensions would be more intuitive. I suppose a dimension of `1` would have been closer to before the change.